lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -45,6 +48,10 @@ class QemuPlatform(Platform):
         SerialConsole,
     ]
 
+    qemu_platform_runbook: QemuPlatformSchema
```

Comment:
This is creating static variables on the class. If you want to predeclare instance variables, it should be done in the `__init__` function.